code/src/components/Container.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components/macro'
+import StartScreen from './StartScreen'
+import LastPage from './LastPage'
+// import { Innerwrapper, Outerwrapper } from './GlobalStyles'
+
+import Loading from './Loading'
+
+const Container = () => {
+  const coordinates = useSelector((store) => store.labyrinth.coordinates)
+  const loadingState = useSelector((store) => store.loading.load)
+
+  if (loadingState) {
+    return (
+      <Loading />
+    )
+  }
+  return (
+    <Outerwrapper coordinates={coordinates}>
+      <Innerwrapper>
+        {coordinates === '' && <StartScreen />}
```

Comment:
These two lines could probably be shortened to {coordinates === '' ? <StartScreen /> : <LastPage />}, or something like that, I think :)